### DISCUSSION TITLE
createSound()

### DISCUSSION THREAD

**Discussion by Refenger** [Leather128/LeathersFunkinEngine#298] [Q&A] [answered]
i have a burst.ogg in mods/MYMOD/sounds/
what do i wtite in createSound() function to use it
i know that arguments are [id, file_path, library, looped] but i don't know what to put in library

**Comment by karstenx10** [comment]
I can ask Leather about that.

**Reply by Refenger** [reply]
pls do it

**Comment by Leather128** [comment, marked as answer]
personally i would use this code `createSound('burst', 'burst', '', false)` (library is blank because it's not in a library)

**Reply by Refenger** [reply]
it doesn't work. attempt to call global 'createSound' (a nil value)

**Reply by Leather128** [reply]
what version of leather engine are you on

**Reply by Leather128** [reply]
also it's running in a function right?

**Reply by Refenger** [reply]
yeah, it worked when i pit it in a function, thx
